sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -73,7 +76,8 @@
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<AttestationResponse>> attestSgxEnclaveWithResponse(AttestSgxEnclaveRequest request) {
```

Comment:
Also, not sure if you plan on adding more convenience layers here but you could make the attestOpenEnclave depend on attestOpenEnclaveWithXXX. This way you will need to add mappers etc to only one method.  It is a preference, seen people do it both ways.

Reply:
This code (and all the rest of the code) will be completely replaced and rewritten with a convenience layer similar to the one in the other languages.

And it is 100% my plan to make the sync version a wrapper around the async wrapper and the xxx api depend on the WithXxx API to simplify the experience.